test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/012_Change_audio_source_from_MOBILE_APP_without_keepContext.lua

```diff
@@ -0,0 +1,86 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) App is RC and Media
+-- 2) App in FULL HMI level
+-- 3) App tries to change audio source from MOBILE_APP with keepContext = false
+-- SDL must:
+-- 1) Change audio source successfully
+-- 2) Change HMI level of App to BACKGROUND
```

Comment:
when app is both remote_control and mobile_projection and media, HMI level will be LIMITED

Reply:
@yang1070 , in our case application is media ( isMediaApplication = true ) and has one HMI REMOTE_CONTROL ( see commonRC.lua, line 8 ) ,so will be set in BACKGROUND.